graph/encoding/digraph6/digraph6.go

```diff
@@ -57,7 +57,12 @@ func Encode(g graph.Graph) Graph {
 
 	var buf strings.Builder
 	buf.WriteByte('&')
-	switch {
+	// digraph6 specifies graphs of order up to 68719476735
```

Comment:
Can this number be expressed using some power of 2, to make it easier to understand?

Reply:
Done